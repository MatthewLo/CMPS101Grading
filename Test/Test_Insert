5
Check list insert
gcc -std=c99 -Wall -c Test_Insert.c
gcc -std=c99 -Wall -o Test_Insert Test_Insert.o ../List.o
./Test_Insert
valgrind --leak-check=full --log-file="memoryLeak" ./Test_Insert
