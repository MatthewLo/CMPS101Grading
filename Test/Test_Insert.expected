0 1 4 9 16 
32 18 8 2 0
0 0 1 1 2 4 3 9 4 16
0
99 0 1 4 9 16 
1
99 0 100 1 4 9 16
1
4
32 18 8 2 99 0
5
32 18 8 2 99 0 100
5
99 0 49 100 50 1 4 9 16
3
